data/css/modules/window/_simple.scss

```diff
@@ -4,12 +4,12 @@
 
     scrollbar {
         padding: 3px;
-        background-color: transparentize(black, 1 - 0.20);
+        background-color: #d6d7d8;
```

Comment:
I think this needs more work, since it doesn't look too good on other presets/themes:

![screenshot from 2018-03-27 15-27-59](https://user-images.githubusercontent.com/481418/37990273-c85d89c0-31d3-11e8-8e33-d180681b775e.png)
![screenshot from 2018-03-27 15-27-38](https://user-images.githubusercontent.com/481418/37990285-d1e047e4-31d3-11e8-9425-dd1d10b41c10.png)

I am going to try to revert these changes and try to extract the core improvements without affecting other presets/themes.